VMEncryption/main/patch/UbuntuPatching.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, logger, distro_info):
         self.resize2fs_path = '/sbin/resize2fs'
         self.umount_path = '/bin/umount'
         self.touch_path = '/usr/bin/touch'
+        self.min_version_online_encryption='22.04'
+        self.support_online_encryption=self.validate_online_encryption_support()
```

Comment:
[vishal]: We don't support online encryption on Ubuntu as the OS disk does not have a separate boot partition.
So based on the above comment support_online_encryption must not be true, whereas 
if CVM and self.validate_online_encryption_support()
     do online data disk encryption.